<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="5" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-05-16T14:31:02 IST" name="Default suite" finished-at="2025-05-16T14:31:09 IST" duration-ms="6896">
    <groups>
    </groups>
    <test started-at="2025-05-16T14:31:02 IST" name="Default test" finished-at="2025-05-16T14:31:09 IST" duration-ms="6896">
      <class name="assignments.Assignment1">
        <test-method is-config="true" signature="setUp()[pri:0, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:02 IST" name="setUp" finished-at="2025-05-16T14:31:02 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addNewPetToStore()[pri:2, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:02 IST" name="addNewPetToStore" finished-at="2025-05-16T14:31:08 IST" duration-ms="5333" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewPetToStore -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:08 IST" name="tearDown" finished-at="2025-05-16T14:31:08 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:08 IST" name="setUp" finished-at="2025-05-16T14:31:08 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="deletePetWithSameID()[pri:5, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:08 IST" name="deletePetWithSameID" finished-at="2025-05-16T14:31:09 IST" depends-on-methods="assignments.Assignment1.addNewPetToStore" duration-ms="1476" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at assignments.Assignment1.deletePetWithSameID(Assignment1.java:165)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePetWithSameID -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:assignments.Assignment1@550ee7e5]" started-at="2025-05-16T14:31:09 IST" name="tearDown" finished-at="2025-05-16T14:31:09 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- assignments.Assignment1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
